themes-default/slim/views/config_search.mako

```diff
@@ -495,18 +652,18 @@ const startVue = () => {
                                     </span>
                                 </label>
                             </div>
-                            <div class="testNotification" id="testNZBget_result">Click below to test</div>
-                                <input class="btn" type="button" value="Test NZBget" id="testNZBget" class="btn test-button"/>
+                            <div class="testNotification" id="testNZBget_result">{{nzb.nzbget.testStatus}}</div>
+                                <input @click="testNzbget" class="btn" type="button" value="Test NZBget" id="testNZBget" class="btn test-button"/>
                                 <input type="submit" class="btn config_submitter" value="Save Changes" /><br>
-                        </div>
+                            </div>
```

Comment:
This part is indented incorrectly:
Both `<input>`s - one less column
This `<div>` back to where it was (closing div for `<div v-show="nzb.method === 'nzbget'" id="nzbget_settings">`)